### ISSUE TITLE
Implement sanity checks for models

### ISSUE THREAD

**Issue by ger-e** [AstraZeneca-NGS/LogMl#12]
For making sure models actually learned something, implement sanity checks such as randomizing model weights, data labels, etc. Could consider using sklearn's [DummyClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.dummy.DummyClassifier.html)

**Comment by pcingola** [comment]
Dummy classifier added to model search.
